### PR TITLE
docs: add sapphire to showcase

### PR DESCRIPTION
**Pull request by favna**
## Motivation

We have started using Docusaurus for the Sapphire project a while back and I figured hey why not add it to the list.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

None.

## Related PRs

None.

### CODE REVIEW

**Comment by favna** [comment]
Uh I have no what's up with the Netlify deploy. It says `Input file contains unsupported image format` but I only added a `.png` image, same as all the other users.

I also don't know what's up with signing the CLA. I would've thought I'd get a bot comment prompting me to sign it. As it stands I have no idea where to go to sign it.

**Comment by armano2** [comment]
even github has an issue with this image, you should try taking new one

https://github.com/facebook/docusaurus/blob/d0a627da1fa1c3dbcff921c84155f5a5663ef7c6/website/src/data/showcase/sapphire.png

```
�PNG

```

**Comment by armano2** [comment]
image

**Comment by favna** [comment]
That's really odd. I used the upload function in github.dev. I'll upload a new one by actually cloning my fork.

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1267,6 +1267,14 @@ const Users: User[] = [
     source: 'https://github.com/mirumee/saleor-docs',
     tags: ['opensource', 'product', 'versioning'],
   },
+  {
+    title: 'Sapphire',
+    description: 'Sapphire is a next-gen object-oriented Discord.js bot framework.',
+    preview: require('./showcase/sapphire.png'),
+    website: 'https://www.sapphirejs.dev',
+    source: 'https://github.com/sapphiredev/website',
+    tags: ['opensource'],
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:1276]:
```suggestion
    tags: ['opensource', 'large'],
```

**Pull request merged**